Source/Editor/Windows/GameCookerWindow.cs

```diff
@@ -343,6 +347,269 @@ public override void Initialize(LayoutElementsContainer layout)
 
                         _buildButton = layout.Button("Build").Button;
                         _buildButton.Clicked += OnBuildClicked;
+                        
+                        // Add emulation options to android tab.
+                        if (_platform == PlatformType.Android)
+                        {
+                            layout.Space(5);
+                            var emulatorGroup = layout.Group("Tools");
+                            var sdkPath = Environment.GetEnvironmentVariable("ANDROID_HOME");
+                            if (string.IsNullOrEmpty(sdkPath))
+                                sdkPath = Environment.GetEnvironmentVariable("ANDROID_SDK");
+                            emulatorGroup.Label($"SDK path: {sdkPath}");
+                            
+                            // AVD and starting emulator
+                            var avdGroup = emulatorGroup.Group("AVD Emulator");
+                            avdGroup.Label("Note: Create AVDs using Android Studio.");
+                            avdGroup.Panel.IsClosed = false;
+                            var refreshAVDListButton = avdGroup.Button("Refresh AVD list").Button;
+                            var avdListGroup = avdGroup.Group("AVD List");
+                            avdListGroup.Panel.IsClosed = false;
+                            var noAvdLabel = avdListGroup.Label("No AVDs detected. Click Refresh.", TextAlignment.Center).Label;
+                            var avdListTree = new Tree(false)
+                            {
+                                Parent = avdListGroup.Panel,
+                            };
+                            refreshAVDListButton.Clicked += () =>
+                            {
+                                if (avdListTree.Children.Count > 0)
+                                    avdListTree.DisposeChildren();
+                                
+                                var processStartInfo = new System.Diagnostics.ProcessStartInfo
+                                {
+                                    FileName = Path.Combine(sdkPath, "emulator", "emulator.exe"),
+                                    Arguments = "-list-avds",
+                                    RedirectStandardOutput = true,
+                                    CreateNoWindow = true,
+                                };
+
+                                var process = new System.Diagnostics.Process
```

Comment:
Have you tried using `FlaxEngine.Platform.CreateProcess` and `FlaxEngine.CreateProcessSettings` to start process via Flax API instead of .NET one? It supports capturing output as array of chars and auto-logging process to the Output Log.

Reply:
Yes I tried this originally, but there is a crash when using the built in Flax platform to start a process when capture output is enabled and I could not figure out why it was crashing.

Reply:
There was also an issue using the built in platform process where the editor just freezes when starting an emulator.

Reply:
Okay fine. Let's use it for now.